src/ApplicationConfigInjectionTrait.php

```diff
@@ -165,24 +162,24 @@ public function injectRoutesFromConfig(array $config = null)
                 continue;
             }
 
+            $methods = Route::HTTP_METHOD_ANY;
             if (isset($spec['allowed_methods'])) {
                 $methods = $spec['allowed_methods'];
                 if (! is_array($methods)) {
-                    throw new Container\Exception\InvalidArgumentException(sprintf(
+                    throw new Exception\InvalidArgumentException(sprintf(
```

Comment:
This is technically a BC break (part of the reason for targeting 2.0 instead of 1.1), but will likely never affect anyone.